### PR TITLE
Update(Readme): Update steps to clone cstor volumes

### PR DESCRIPTION
**Pull request by payes**

### CODE REVIEW

File: deploy/snapshot-class.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+kind: VolumeSnapshotClass
+apiVersion: snapshot.storage.k8s.io/v1alpha1
+metadata:
+  name: csi-cstor
```
Comment (prateekpandey14) [inline deploy/snapshot-class.yaml]: nit: `csi-cstor-snapshotclass` ?

File: README.md [hunk 2]
```diff
@@ -192,3 +192,35 @@ be seen in running state.
 It is internally a two step process for volumes containing a file system:
 1. Volume expansion
 2. FileSystem expansion
+
+### Clone a cstor volume using OpenEBS CSI Driver
```
Comment (prateekpandey14) [inline README.md]: nit: `Snapshot and Clone CStor Volume using OpenEBS CSI Driver`

File: README.md [hunk 3]
```diff
@@ -192,3 +192,35 @@ be seen in running state.
...
+```
+kubectl apply -f https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/snapshot.yaml
+```
+3. Verify that the snapshot has been created successfully:
+```
+kubectl get volumesnapshots.snapshot 
+NAME            AGE
+demo-snapshot   3d1h
+```
+4. Create the cloned PV using the above Snapshot by updating and modifying the following yaml:
```
Comment (prateekpandey14) [inline README.md]: nit: PV --> PVC ?

File: deploy/snapshot.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+apiVersion: snapshot.storage.k8s.io/v1alpha1
+kind: VolumeSnapshot
+metadata:
+  name: demo-snapshot
+spec:
+  snapshotClassName: csi-cstor
```
Comment (prateekpandey14) [inline deploy/snapshot.yaml]: Same as above name `csi-cstor-snapshotclass` just to differentiate with normal storageclass

File: README.md [hunk 5]
```diff
@@ -93,92 +93,92 @@ openebs-csi-node-56t5g     2/2     Running   0          6m13s
...
+through various components.
 
-1. Create PVC with Storage Class referring to OpenEBS CSI Driver.
+1. Create PVC with Storage Class referring to OpenEBS CStor CSI Driver.
 
 2. Kubernetes will pass the PV creation request to the OpenEBS
    CSI Controller service via `CreateVolume()`, as this controller
    registered with Kubernetes for receiving any requests related to
-   `openebs-csi.openebs.io`  
+   `openebs-csi.openebs.io`
```
Comment (prateekpandey14) [inline README.md]: nit: `cstor.csi.openebs.io`

File: README.md [hunk 6]
```diff
@@ -1,47 +1,47 @@
...
 
-CSI driver implementation for OpenEBS storage engines. 
+CSI driver implementation for OpenEBS CSTOR storage engine.
 
 ## Project Status
 
 This project is under active development and considered to be in Alpha state.
 
-The current implementation only supports provisioning, de-provisioning and expansion of cStor Volumes. 
+The current implementation only supports provisioning, de-provisioning and expansion of cStor Volumes.
```
Comment (kmova) [inline README.md]: Should this be changed to specify snapshot and clone?

File: README.md [hunk 7]
```diff
@@ -1,47 +1,47 @@
-# OpenEBS CSI Driver
+# OpenEBS CStor CSI Driver
 
-CSI driver implementation for OpenEBS storage engines. 
+CSI driver implementation for OpenEBS CSTOR storage engine.
```
Comment (kmova) [inline README.md]: n_: CStor

File: README.md [hunk 8]
```diff
@@ -1,47 +1,47 @@
...
 
-The current implementation only supports provisioning, de-provisioning and expansion of cStor Volumes. 
+The current implementation only supports provisioning, de-provisioning and expansion of cStor Volumes.
 
 ## Usage
 
 ### Prerequisites
 
-Before setting up OpenEBS CSI driver make sure your Kubernetes Cluster 
+Before setting up OpenEBS CStor CSI driver make sure your Kubernetes Cluster
```
Comment (kmova) [inline README.md]: Add one more note or pre-req .. about CStor CSI Driver operates on the cStor Pools created using the new Schema called CSPC. Provide a link to the design to this new CSPC schema.

**Pull request merged**